DeckSwipe/Assets/Resources/BillingMode.json

```diff
@@ -0,0 +1 @@
+{"androidStore":"GooglePlay"}
```

Comment:
Is this created automatically when opening with the newer editor?

Reply:
That's right. Even if you don't the Android module installed for the editor version.